hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -107,6 +107,58 @@
       '''
     }
   ],
+  features: [
+    { name: "SPIHOST.USECASE.SERIALNORFLASH",
+      desc: '''
+        The SPI host block should be able to talk to serial NOR flash devices.
+      '''
+    },
+    { name: "SPIHOST.USECASE.PASSTHROUGH",
+      desc: '''
+        The SPI host block should be able to talk to serial NOR flash devices.
```

Comment:
I think this text needs tweaking: it's the same as the feature above :-)

Reply:
Thanks, should be resolved.